addOns/reports/src/test/java/org/zaproxy/addon/reports/ExtensionReportsUnitTest.java

```diff
@@ -552,22 +547,19 @@ void shouldGenerateExpectedReport(String templateName)
         assertThat(cleanReport(report), is(equalTo(cleanReport(expected))));
     }
 
-    private static void generateTestFile(String templateName)
-            throws IOException, DocumentException {
+    private static void generateTestFile(String templateName) throws Exception {
 
-        Template template =
-                new Template(
-                        new File(
-                                "src/main/zapHomeFiles/reports/"
-                                        + templateName
-                                        + "/template.yaml"));
+        Template template = new Template(new File(getResourceTemplateYaml(templateName)));
         generateReportWithAlerts(
                 template,
                 new File(
-                        "src/test/resources/org/zaproxy/addon/automation/resources/basic-"
```

Comment:
- In this case it needs to be kept the hardcoded path, it's creating new files in that directory (while some of them already exist, new reports wouldn't).
 - Note that the `automation` is a typo should be `reports` (probably copy/paste).